### DISCUSSION TITLE
experimentaL version EFB

### DISCUSSION THREAD

**Discussion by dbryant69** [flybywiresim/aircraft#4424] [Help] [answered]
I cannot get the EFB to link with the mcdu or to acttivate a simbrief flt plan that has been loaded into the mcdu. I have entered my simbrief userid whick the EFB had accepted. is this option not active yet. I have downloaded the latest update as of this morning so am up to date

**Comment by Annolago** [comment]
Works perfectly fine for me. Have you loaded the MCDU from Simbrief before using the EFB?

**Reply by dbryant69** [reply]
Yes loads into the MCDU perfectly. Then I add my userid into the EFB and click on the from simbrief button and nothing happens. Not at my computer at the moment so don’t know what experimental version I have but I updated to the latest version yesterday (13th)

**Reply by dbryant69** [reply]
Also while I think of it, I seem to have to log into the EFB each time unlike the MCDU which holds the login

**Reply by dbryant69** [reply]
See the below suggestion from ExampleWasTaken.. I was using the wrong login. but thanks for taking the trouble to get back to me

**Comment by ExampleWasTaken** [comment, marked as answer]
I am not sure what exactly you mean but here are a few things that I think could help you:
1. The `Link MCDU` option in the EFB is not yet functional.
2. To load a SimBrief flight plan, enter your SimBrief username in the text field of the company page of the EFB. (Tip: In order to prevent your input from affecting the simulator, you can either pop out the EFB using `Right Alt + Click` and enter it in the EFB window, or you can write it in your text editor and paste it into the EFB using `Ctrl+V`.
3. The EFB requires your SimBrief username, not your SimBrief ID.
4. The EFB does not store your SimBrief username at the moment. An implementation for that is planned but not yet implemented.

**Reply by dbryant69** [reply]
You spotted my deliberate mistake, I was trying to log in using my userid. Now I'm using my username it works perfectly. Thanks for setting me right

**Reply by ExampleWasTaken** [reply]
Glad I could help 👍

**Reply by docjonny** [reply]
Original poster does present good question. MCDU holds the SimBrief username even after closing MSFS to desktop. However, each time EFB is open one has to enter SimBrief username on company page. How to get MSFS with A320NX mod to save the username permanently?

**Reply by ExampleWasTaken** [reply]
@docjonny see point 4 of my previous post

**Reply by docjonny** [reply]
Roger that. Slow reader or skipping the obvious. Clear skies and fair winds friend!

**Reply by tracernz** [reply]
> Original poster does present good question. MCDU holds the SimBrief username even after closing MSFS to desktop. However, each time EFB is open one has to enter SimBrief username on company page. How to get MSFS with A320NX mod to save the username permanently?

FlyPadOSv2 coming soon.
